### PR TITLE
Remove Google Analytics

### PR DESCRIPTION
**Pull request by jwnrt**
Removes Google Analytics from the site and brings fonts into this repo so clients don't have to connect to any remote servers. Also removes the _Cookies_ section from the privacy policy since we no longer store any.

The GA cookies already stored in browsers will disappear once they expire or are cleared manually. Alternatively we could serve some JavaScript that updates them to expire immediately.

I've tested the font change by flicking between the old and new renders and don't see any differences on all the main pages plus the blogs that I checked.

Closes https://github.com/lowRISC/lowrisc-web/issues/9.

### CODE REVIEW

File: site/content/privacy-policy/index.html [hunk 1]
```diff
@@ -15,26 +15,6 @@ <h2> General website information</h2><p>
...
-  A cookie called <code>_gat</code> is used by Google Analytics to throttle the request rate.
-
-
-</P>
-  <p>For more information see the <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage">Google Analytics cookies page</a>.
-
-</P>
-  <h3> How to control cookies</h3>
-<p>  You can control and/or delete cookies as you wish – for details, see <a href="aboutcookies.org">aboutcookies.org</a>. You can delete all cookies that are already on your computer and you can set most browsers to prevent them from being placed. If you do this, however, you may have to manually adjust some preferences every time you visit a site and some services and functionalities may not work.
-</P>
```
Comment (HU90m) [inline site/content/privacy-policy/index.html:37]: Flagging this for @mundaym .

It is probably fine to remove, but James and I weren't sure. There aren't any cookies used by the site anymore. There will of course be cookies on the client until they expire.

File: src/scss/app.scss [hunk 2]
```diff
@@ -4,10 +4,36 @@
 
 @import "variables";
 
-@import url('https://fonts.googleapis.com/css?family=Exo+2:400,700|Source+Sans+Pro:400,700&display=swap');
-
```
Comment (HU90m) [inline src/scss/app.scss:8]: 🤩

**Review by HU90m — APPROVED** [review]
This looks great!

File: site/content/privacy-policy/index.html (hunk 1 above)
Comment (mundaym) [reply site/content/privacy-policy/index.html:37]: LGTM, thanks both

**Review by mundaym — APPROVED** [review]

**Pull request merged**